rules/no-await-in-loop.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * @fileoverview Rule to disallow uses of await inside of loops.
+ * @author Nat Mote
+ */
+"use strict";
+
+// Node types which are considered loops.
+var loopTypes = new Set([
```

Comment:
Do we have a target minimum version? I can use an `Object` instead of necessary.